common/src/unifyfs_configurator.h

```diff
@@ -71,6 +71,7 @@
     UNIFYFS_CFG_CLI(unifyfs, mountpoint, STRING, /unifyfs, "mountpoint directory", NULL, 'm', "specify full path to desired mountpoint") \
     UNIFYFS_CFG(client, max_files, INT, UNIFYFS_MAX_FILES, "client max file count", NULL) \
     UNIFYFS_CFG(client, flatten_writes, BOOL, on, "flatten writes", NULL) \
+    UNIFYFS_CFG(client, local_extents, BOOL, off, "track extents to read back local data", NULL) \
```

Comment:
Mind adding this to the client section in the configuration documentation as well?

https://github.com/LLNL/UnifyFS/blob/dev/docs/configuration.rst#unifyfsconf